tools/travis-upload-to-gdrive.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+source tools/travis-helpers.sh
+
+set -euo pipefail
+
+echo "Uploading test results to google drive"
+export GOPATH="/tmp/go"
+go get github.com/prasmussen/gdrive
+
+echo "${GDRIVE_SERVICE_ACCOUNT_CREDENTIALS}" | base64 --decode > /tmp/serviceAccountCredentials
+
+gdrive() {
+    /tmp/go/bin/gdrive --config /tmp --service-account serviceAccountCredentials "$@"
+}
+
+gdrive_get_id() {
```

Comment:
comment why do we need ids.
And generally how grive uses IDs instead of file names to address stuff.
An example of usage.

Same comment for all functions defined here.
Basically, I don't know what is $1 and $2 until I read code.
So, args and output should be documented.